apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/sql/SqlStore.java

```diff
@@ -0,0 +1,17 @@
+package com.apollographql.android.cache.normalized.sql;
+
+
+import com.apollographql.android.cache.normalized.CacheStore;
+import com.apollographql.android.cache.normalized.Record;
+
+import javax.annotation.Nullable;
+
+public class SqlStore extends CacheStore {
```

Comment:
to me there were 2 parts to `InMemoryStore` the part that implements the interfaces and then part that actually does the storing which is implemented as `RecordSet`. I tried to keep this consistent so that `StorePersister` deals with the data access while `SqlStore` does the stuff like parsing.